pkg/instrumentation/auto/config.go

```diff
@@ -10,6 +10,7 @@ import "github.com/aws/amazon-cloudwatch-agent-operator/pkg/instrumentation"
 type AnnotationConfig struct {
 	Java   AnnotationResources `json:"java"`
 	Python AnnotationResources `json:"python"`
+	DotNet AnnotationResources `json:"dotnet"`
```

Comment:
can someone confirm if this is all thats needed for auto annotation? or if there's anything else that needs to be done here

Reply:
I tried doing auto annotation with this added but it didn't seem to work... maybe my syntax is wrong:

`- --auto-annotation-config={"java":{"daemonsets":[],"deployments":[],"namespaces":["default"],"statefulsets":[]},"python":{"daemonsets":[],"deployments":[],"namespaces":[],"statefulsets":[]},"dotnet":{"daemonsets":[],"deployments":["default/sample-deployment"],"namespaces":["default"],"statefulsets":[]}
`

Reply:
So the issue is your missing "instrumentation.TypeDotNet" when the autoAnnotationMutators var is being created.
Suggestion for line 240
```suggestion
	//Original code: 
	autoAnnotationMutators := auto.NewAnnotationMutators(
				mgr.GetClient(),
				mgr.GetAPIReader(),
				logger,
				autoAnnotationConfig,
				instrumentation.NewTypeSet(
					instrumentation.TypeJava,
					instrumentation.TypePython,
				),
			)
	//Suggested Code:
	autoAnnotationMutators := auto.NewAnnotationMutators(
			mgr.GetClient(),
			mgr.GetAPIReader(),
			logger,
			autoAnnotationConfig,
			instrumentation.NewTypeSet(
				instrumentation.TypeJava,
				instrumentation.TypePython,
				instrumentation.TypeDotNet, //this needs to be added
			),
		)
```

Reply:
thank you for fixing this